tests/test_cli_export_e2e.py

```
@@ -0,0 +1,319 @@
+
+    def setUp(self):
+        self._prior = os.environ.get("XDG_STATE_HOME")
+
+    def tearDown(self):
+        if self._prior is None:
+            os.environ.pop("XDG_STATE_HOME", None)
+        else:
+            os.environ["XDG_STATE_HOME"] = self._prior
+
+    def test_uses_xdg_state_home_when_set(self):
+        os.environ["XDG_STATE_HOME"] = "/tmp/some-xdg-root"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/some-xdg-root"

(S108)